tests/Tokenizer/Transformer/TransformersTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\tests\Tokenizer\Transformer;
+
+use PhpCsFixer\Tokenizer\TransformerInterface;
+use PhpCsFixer\Tokenizer\Transformers;
+
+/**
+ * @internal
+ *
+ * @author SpacePossum
+ */
+final class TransformersTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @param TransformerInterface $transformer
+     *
+     * @dataProvider provideTransformers
+     */
+    public function testTransformersAreFinal(TransformerInterface $transformer)
```

Comment:
maybe sth like this:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/tests/ProjectCodeTest.php#L179
but for src ?

then, i guess we will need few exceptions that will be marked as todo for 3.x line